vendor/github.com/opencontainers/selinux/go-selinux/selinux.go

```diff
@@ -498,14 +580,14 @@ func DupSecOpt(src string) []string {
 		con["level"] == "" {
 		return nil
 	}
-	return []string{"label=user:" + con["user"],
-		"label=role:" + con["role"],
```

Comment:
@cpuguy83 selinux hasn't any notion of `label=` prefixed labels. The label is just the label, not `label=label` <- this is something docker did afacit, nothing related to selinux /cc @rhatdan